packages/ra-ui-materialui/src/button/PrevNextButtons.tsx

```diff
@@ -108,7 +115,7 @@ export const PrevNextButtons = <RecordType extends RaRecord = any>(
 
     if (isLoading) {
         return (
-            <Box minHeight={theme => theme.spacing(4)}>
```

Comment:
I can't keep consistent height between the empty / loading / loaded versions unless I switch to pixels